code/modules/antagonists/traitor/contractor/datums/syndicate_contract.dm

```diff
@@ -473,6 +473,14 @@
 		M.EyeBlurry(35 SECONDS)
 		M.AdjustConfused(35 SECONDS)
 
+		for(var/mob/living/simple_animal/hostile/guardian/G in GLOB.alive_mob_list)
+			if(G.summoner == M)
+				M.remove_guardian_actions()
+				to_chat(G, "<span class='danger'>You feel your body ripped to shreds as you're forcibly removed from your summoner!</span>")
+				to_chat(G, "<span class='warning'>You feel some part of you missing, you're not who you used to be...</span>")
```

Comment:
```suggestion
				to_chat(M, "<span class='warning'>You feel some part of you missing, you're not who you used to be...</span>")
```
assume this is to be sent to the mob vs the guardian

Reply:
how do i send the walter white falling gif on discord

Reply:
https://images-ext-2.discordapp.net/external/6L6UhsB8cyRAxIHNMgdWk_OME176bSzIDjy3jzsGjWg/https/media.tenor.com/nicIRWJkKIwAAAPo/walter-white-walter.mp4